src/assets/styles/mixins/cards.scss

```diff
@@ -37,8 +37,6 @@
       flex-wrap: wrap;
       gap: $gap;
       height: 100%;
```

Comment:
Why were these numbers removed? This breaks card layout in details.

Reply:
returned